### PR TITLE
CI: fix OpenBSD CI test

### PR DESCRIPTION
**Pull request by fingolfin**
Attempt to fix OpenBSD CI tests. But I am stumbling the dark here. Perhaps @dimpase can help?

### CODE REVIEW

**Comment by dimpase** [comment]
Is it autoconf-related?
OpenBSD needs explicit setting of autoconf and automake versions via environment variables.
I.e. GAP's .`/autogen needs
```
export AUTOCONF_VERSION=2.71
```

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -382,7 +382,7 @@ jobs:
           envs: 'AUTOCONF_VERSION'
```
[the remark this answers is not in the document]
Comment (dimpase) [reply .github/workflows/CI.yml:382]: It should be `autoconf-2.71p0`  I think.

(`pkg_add` is interactive; `pkg_add autoconf` enters a dialog with a menu of versions to choose from; the only 2.71 one is as above)

[the remark this answers is not in the document]
Comment (dimpase) [reply .github/workflows/CI.yml:382]: there is an `autoconf-2.72p0` too, it seems.

**Review by dimpase — APPROVED** [review]
OK, with my latest commit, OpenBSD CI works.

**Comment by fingolfin** [comment]
Thanks Dima!

(Just wondering, out of curiosity: do you know why it worked before and now needs the `p0`?)

**Comment by dimpase** [comment]
> (Just wondering, out of curiosity: do you know why it worked before and now needs the `p0`?)
They renamed the package as they applied a patch:
https://github.com/openbsd/ports/commit/953c48a60e99f043285fad0bba106e52c227073d

**Pull request merged**